codecov.yaml

```diff
@@ -0,0 +1,3 @@
+ignore:
+  - "./ThirdParty"
+  - "./*Tests"
```

Comment:
Also, with this all the folders containing the tests are ignored for the code coverage results. That is more similar to the way bb shows the coverage results.

@st3fan do you prefer to keep these files so that they count for the coverage or better to ignore them as in this PR. Results could be seen [here](https://codecov.io/gh/mozilla-mobile/firefox-ios/pull/2678/tree)

Plese let me know other files you would like to ignore/show if any. Thanks!